lambda/src/main/scala/pricemigrationengine/handlers/AmendmentHandler.scala

```diff
@@ -99,16 +104,20 @@ object AmendmentHandler extends CohortHandler {
             )
           )
         )
+
       whenDone <- Clock.instant
-    } yield SuccessfulAmendmentResult(
-      item.subscriptionName,
-      startDate,
-      oldPrice,
-      newPrice,
-      estimatedNewPrice,
-      newSubscriptionId,
-      whenDone
-    )
+    } yield {
+      println(s"update: ${update}")
```

Comment:
You can always do `.debug("update")` on line 84 instead for an ad hoc inspection
https://zio.dev/guides/tutorials/debug-a-zio-application/#debugging-a-zio-application-using-debug-effect

Reply:
Thanks @kelvin-chappell !